server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -124,17 +133,21 @@ public Content render(
   }
 
   private FormTag renderSearchForm(
-      ProgramDefinition program, ButtonTag downloadButton, RenderFilterParams filterParams) {
+      ProgramDefinition program,
+      ImmutableList<String> allApplicationStatuses,
```

Comment:
This won't show program status unless an application has it right?  So initially no Statuses would be present right?

Not a blocker but let's run it by Raha too

Reply:
This is a list of all unique application statuses for all versions of a program and isn't directly tied to what applications have the statuses set. Put another way, the source of this information comes from ProgramDefinitions, not Applications.

I renamed to `allPossibleProgramApplicationStatuses` in an effort to clarify this.

As such, I don't think there's anything we need to run by UX since I imagine we'd want to show all statuses that could be selected, not just the ones that currently are set on an application.